text/0124-documents-api.md

```diff
@@ -161,6 +177,7 @@ Gives the total number of documents that can be browsed in the related index.
 - 🔴 Sending a value with a different type than `Integer` or `null` for `offset` will return a [invalid_document_offset](0061-error-format-and-definitions.md#invalid_document_offset) error.
 - 🔴 Sending a value with a different type than `Integer` or `null` for `limit` will return a [invalid_document_limit](0061-error-format-and-definitions.md#invalid_document_limit) error.
 - 🔴 Sending a value with a different type than `String` or `null` for `fields` will return a [invalid_document_fields](0061-error-format-and-definitions.md#invalid_document_fields) error.
+- 🔴 Sending a value with a different type than `filter` or `null` for `filter` will return a [invalid_document_filter](0061-error-format-and-definitions.md#invalid_document_filter) error.
```

Comment:
What does means "Sending a value with a different type than `filter`?"

Reply:
Oops, I forgot to edit this one, I wanted to copy-paste the super long `string or array of string or array of array of string` thingy, and I wrote `filter` as a placeholder in the meantime :grimacing: 

It could be interesting to define somewhere in the spec what a filter is as a query parameter or a json body and then use it everywhere because it doesn't help the readability to copy-paste this long series of types everywhere.